src/main/java/games/strategy/util/memento/MementoImporter.java

```diff
@@ -0,0 +1,19 @@
+package games.strategy.util.memento;
+
+/**
+ * Imports a memento to its originator.
+ *
+ * @param <T> The type of the memento originator.
+ */
+public interface MementoImporter<T> {
+  /**
+   * Imports the specified memento and returns the originator.
+   *
+   * @param memento The memento to be imported; must not be {@code null}.
+   *
+   * @return The memento originator; never {@code null}.
+   *
+   * @throws MementoImportException If an error occurs while importing the memento.
```

Comment:
nit: `MementoImportException` seems to imply an exception during memento import. "error occurs while importing the memento." is rephrasing, not really adding much, and that is still just restating the same name of the exception.

With respect, I think we could save a bit here by skipping seemingly obvious commentary in javadocs. What do you think, do you find this type of commentary useful? Or are we writing it mostly because it's available and feels complete?